docs/reference-guides/monitoring-v2-configuration/receivers.md

```diff
@@ -211,6 +213,87 @@ url http://rancher-alerting-drivers-sachet.ns-1.svc:9876/alert
 
+## Telegram
+
+### Enabling the Telegram Receiver for Rancher Managed Clusters
+
+The Telegram receiver is not a native receiver and must be enabled before it can be used. You can enable the Telegram receiver for a Rancher managed cluster by going to the Apps page and installing the rancher-alerting-drivers app with the Telegram option selected.
+
+1. In the upper left corner, click **☰ > Cluster Management**.
+1. On the **Clusters** page, go to the cluster where you want to install `rancher-alerting-drivers` and click **Explore**.
+1. In the left navigation bar, click on **Apps**.
+1. Click the **Alerting Drivers** app.
+1. Click on **Install**.
+1. In the page that opens next, make sure that **Enable SMS** checkbox is selected.
+1. Take note of the namespace used as it will be required in a later step.
+
+### Configure a PrometheusRule
+
+To test our Telegram setup, let's create a **PrometheusRule** that will fire all the time. 
+
+:::caution NOTE
+This rule is suggested only for the purpose of testing if Telegram alerts work as expected. Do not leave it on beyond the purpose of testing.
+:::
+
+1. Click on **Monitoring**.
+2. Click on **Advanced**.
+3. Click on **PrometheusRules** and click on **Create** to create a new rule.
+4. Select a namespace to place the rule in; name the rule appropriately; set the group name to `test`. We will use this value later while creating a **Route** in our **AlertManagerConfig**.
+5. Under **Alerting Rules** click on **Add**.
+6. Set an appropriate **Alert Name**.
+7. Enter the following PromQL Expression - `vector(1)`. This will trigger alert immediately and continuously, and is only recommended to test things out.
+8. Under **Labels** add a new one by clicking on **Add Label**. Entre the key `test` and value `alert`. This key-value pair will also be used later.
+
+### Configure an AlertManagerConfig
+
+Next up, configure an **AlertManagerConfig** that will contain the **Receiver** and **Route** configuration to be used for the **PrometheusRule** we created above.
+
+#### Create an AlertManagerConfig
+
+1. Click on **Monitoring**; then click on **Alerting**, and open **AlertManagerConfigs**.
+2. Click on **Create**
+
+#### Create a Receiver in AlertManagerConfig
+
+1. Choose a namespace from the dropdown and set an appropriate name. Click **Create**.
+2. Open the newly created **AlertManagerConfig**. Look for three dots near the top-right of the page and click **Edit Config**.
+3. Create a new Receiver by clicking **Add Receiver**.
+4. Select **Webhook** from the list that shows up on the page **Create Receiver in AlertmanagerConfig**. 
+5. Name the webhook. Then click **Add Webhook**. In the **Select Webhook Type** dropdown select **SMS**. This will automatically populate the **Target** field as `http://rancher-alerting-drivers-sachet.cattle-monitoring-system.svc:9876/alert`. If you installed the **Alerting Drivers** in a namespace other than `cattle-monitoring-system`, above URL will be reflect that. 
```

Comment:
```suggestion
1. Choose a namespace from the dropdown and set an appropriate name. 
1. Click **Create**.
1. Open the newly created **AlertManagerConfig** and click **⋮ > Edit Config**.
1. Click **Add Receiver**.
1. Select **Webhook** from the list on the **Create Receiver in AlertmanagerConfig** page. 
1. Name the webhook, and click **Add Webhook**. 
1. In the **Select Webhook Type** dropdown, select **SMS**. This will automatically populate the **Target** field as `http://rancher-alerting-drivers-sachet.cattle-monitoring-system.svc:9876/alert`. If you installed the **Alerting Drivers** in a namespace other than `cattle-monitoring-system`, the target URL will reflect that. 
```

SMS or Telegram?